### PR TITLE
Feature/experiment pv data

### PR DESCRIPTION
**Pull request by C3MO**

### CODE REVIEW

File: rtp_backend/apps/utilities/http_status_codes.py [hunk 1]
```diff
@@ -10,3 +10,4 @@
 
 # 5XX (server errors)
 NOT_IMPLEMENTED = 501
+INTERNAL_SERVER_ERROR = 500
```
Comment (devmarcstorm) [inline rtp_backend/apps/utilities/http_status_codes.py:13]: Please enter in ascending order

File: docs/experiment_pv_data.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+*In the database there is a table users_to_experiments, which handles the relationship between users and experiments.*
+
+## Endpoints
+
+### Get a list of experiments
+
+- Endpoint: `experiments/data`
+- Method: `POST`
+
+*Send an **admin** access token in the "x-access-tokens" header!*
```
Comment (devmarcstorm) [inline docs/experiment_pv_data.md]: Not admins! All Users in the Experiment have access.

Comment (C3MO) [reply docs/experiment_pv_data.md]: Done!

File: rtp_backend/apps/experiment_pv_data/helper_functions.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        if only_validate:
+            return True
+
+        return {"experiment": experiment.to_dict(), "process_variables_data": data}
+    except:
+        if only_validate:
+            return False
+        
+        return make_response({"errors": ["Could not get process variable data from archiver."]}, 500)
```
Comment (devmarcstorm) [inline rtp_backend/apps/experiment_pv_data/helper_functions.py]: Please use status.BAD_GATEWAY or status.INTERNAL_SERVER_ERROR

File: rtp_backend/apps/experiment_pv_data/helper_functions.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+from datetime import datetime, timedelta, tzinfo
+from http.client import BAD_GATEWAY
```
Comment (devmarcstorm) [inline rtp_backend/apps/experiment_pv_data/helper_functions.py]: BAD_GATEWAY not used

**Review by devmarcstorm — APPROVED** [review]

**Pull request merged**